tests/api/test_discount.py

```diff
@@ -271,28 +275,34 @@ def test_create_voucher_with_existing_voucher_code(
 def test_update_voucher(staff_api_client, voucher, permission_manage_discounts):
     query = """
     mutation  voucherUpdate($code: String,
-        $discountValueType: DiscountValueTypeEnum, $id: ID!) {
+        $discountValueType: DiscountValueTypeEnum, $id: ID!,
+        $applyOncePerOrder: Boolean) {
             voucherUpdate(id: $id, input: {
-                code: $code, discountValueType: $discountValueType}) {
+                code: $code, discountValueType: $discountValueType,
+                applyOncePerOrder: $applyOncePerOrder
+                }) {
                 errors {
                     field
                     message
                 }
                 voucher {
                     code
                     discountValueType
+                    applyOncePerOrder
                 }
             }
         }
     """
     # Set discount value type to 'fixed' and change it in mutation
+    apply_once_per_order = not voucher.apply_once_per_order
```

Comment:
You put this line between line's comment and line. You should keep them together or remove the comment if it is not needed